### DISCUSSION TITLE
How do I tag text that will be customized during the build process?

### DISCUSSION THREAD

**Discussion by mfgorsuch** [squidfunk/mkdocs-material#6374] [Q&A]
We are kicking off a new project where we need to be able to build separate help systems where only the product name changes. In other words, I need to build one help system for the product branded as "A", and one help system for the product branded as "B", but all other text in the help systems is identical.
How can I tag product names so that we can customize the product names during the build process?

**Comment by alexvoss** [comment]
This does sound like a case for the [MkDocs macros plugin](https://mkdocs-macros-plugin.readthedocs.io/en/latest/)?
